MetaMask/metamask-extension
# UX Multichain: Added extra margin for native token

This Pr is to add extra 8px gap between the native token and assets tab.

Fixes #18987 

## Screenshots/Screencaps

### After

![Screenshot 2023-05-03 at 6 18 33 PM](https://user-images.githubusercontent.com/39872794/235920541-8324ee42-89c0-42a7-8951-507f8b6822d1.jpg)



## Manual Testing Steps

- Go to the Assets Tab
- Check the spacing between the native token and tab header is "8px"

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
